editor.planx.uk/src/pages/FlowEditor/components/Team/tests/TeamMembers.addNewEditor.test.tsx

```diff
@@ -1,33 +1,85 @@
-/* eslint-disable jest/expect-expect */
-import { screen, within } from "@testing-library/react";
+import { screen, waitFor, within } from "@testing-library/react";
 
 import { setupTeamMembersScreen } from "./helpers/setupTeamMembersScreen";
+import { userEntersInput } from "./helpers/userEntersInput";
 
 jest.mock("lib/featureFlags.ts", () => ({
   hasFeatureFlag: jest.fn().mockReturnValue(true),
 }));
+jest.mock("pages/FlowEditor/lib/store/team.ts", () => ({
+  teamStore: jest.fn().mockImplementation(() => ({
+    fetchCurrentTeam: jest.fn().mockResolvedValue({
+      team_id: 1,
+      role: "teamEditor",
+      user_id: 1,
+      __typename: "team_members",
+    }),
+  })),
+}));
+jest.mock("pages/FlowEditor/components/Team/queries/createUser.tsx", () => ({
+  createUser: jest.fn().mockResolvedValue({
+    id: 1,
+    email: "mickeymouse@email.com",
+    first_name: "Mickey",
+    last_name: "Mouse",
+    is_platform_admin: false,
+    __typename: "users",
+  }),
+}));
+jest.mock("pages/FlowEditor/components/Team/queries/addUserToTeam.tsx", () => ({
+  addUserToTeam: jest.fn().mockResolvedValue({
+    team_id: 1,
+    role: "teamEditor",
+    user_id: 1,
+    __typename: "team_members",
+  }),
+}));
```

Comment:
I know this is a lot of mocks! I'm looking forward to discussing testing in general, but this is more how I've been used to doing testing in the past. 

Ideally I'd like to find a way to extract these away so the test file is cleaner. ✨